quesma/quesma/functionality/bulk/bulk.go

```
@@ -95,7 +96,13 @@ func Write(ctx context.Context, defaultIndex *string, bulk types.NDJSON, lm *cli
 			if len(cfg.IndexConfig[indexName].Override) > 0 {
 				indexName = cfg.IndexConfig[indexName].Override
 			}
-			return lm.ProcessInsertQuery(ctx, indexName, documents)
+			nameFormatter, err := registry.TableColumNameFormatterFor(indexName, cfg, nil)
+			if err != nil {
+				logger.Error().Msgf("Error getting table column name formatter for index %s: %v", indexName, err)
```

We should return an error here

---

Done